Adyen/Core/Core Protocols/PresentableComponent.swift

```diff
@@ -48,33 +45,35 @@ public protocol PresentableComponent: Component {
     var viewController: UIViewController { get }
     
     /// Indicates whether Component implements a custom Navigation bar.
+    @_spi(AdyenInternal)
     var navBarType: NavigationBarType { get }
 }
 
-/// :nodoc:
+/// A component that provides a view controller for the shopper to fill payment details.
 public extension PresentableComponent {
     
-    /// :nodoc:
+    /// Provides default value of `false`
     var requiresModalPresentation: Bool { false }
```

Comment:
No need to comment here, because it will pick up original comment from `PresentableComponent `